.all-contributorsrc

```diff
@@ -274,6 +274,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "Exspiravit",
+      "name": "Randy Real",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/22334680?v=4",
+      "profile": "https://exspiravit.ga/,
+      "contributions": [
+        "code"
+      ]
     }
```

Comment:
Please remove this we would add you as a contributor when the PR get's merged

Reply:
ok i remove it :+1: